src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader_writer_shared/EventLogFileHandler.java

```diff
@@ -86,11 +86,11 @@ public void writeTmpFileWithPrivilege(List<Event> dataEntries, long epoch) {
         writeInternal(out, data);
       }
     } catch (IOException e) {
-      e.printStackTrace();
       LOG.error(
           "Error writing entry '{}'. Cause: {}",
           currEntry == null ? "NOT_INITIALIZED" : currEntry.key,
-          e.getMessage());
+          e.getMessage(),
+          e);
```

Comment:
Will we print the `Cause` twice  ? Once as part of `e.getMessage()` and then with `e` ?

Reply:
Right. Fixed this.